js/scenes/AccountSettings/AccountSettings.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  Image,
+  TextInput,
+  ScrollView,
+  TouchableOpacity
+} from 'react-native';
+import SurveyButton from '../../components/SurveyButton/';
+import { styles } from './styles';
+
+const AccountSettings = ({ handleImageUpload, imageUrl, handleEmail, emailField, handlePassword, passwordField, handleFullname, fullnameField }) => {
+  return (
```

Comment:
If there's no variable declaration, you don't need a `return` statement.

Reply:
Please ignore the AccountSettings components as it's still being worked on (included in the PR by accident during the squashing of commits).